readme.md

```diff
@@ -82,7 +100,15 @@ var Panda = function(name, age) {
 
 Your Answer:
 ```text
-
+eat_bamboo: function() {
+  var self = this;
```

Comment:
If self is assigned where it is used, then it is irrelevant; `self` _is_ `this`.  You might as well just use `this` and avoid the complexity.